src/components/calculator.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className='calculator-container'>
+        <div className='result-pad'>0</div>
+        <div className='buttons'>
+            <button type='button'>AC</button>
+            <button type='button'>+/-</button>
+            <button type='button'>%</button>
+            <button type='button'>/</button>
```

Comment:
- [ ] According to the design this is not the given `division` sign. It should be this &#xF7;. You can use this entity code, instead of `/`, if you don't have it: `&#xF7;`